frontend/src/metabase/lib/redux/utils.unit.spec.js

```diff
@@ -125,9 +125,7 @@ describe("Metadata", () => {
       });
 
       try {
-        const dataFail = await fetchData(argsFail).catch(error =>
-          console.log(error),
-        );
+        const dataFail = await fetchData(argsFail);
```

Comment:
I ran this test and saw that there's no change in either the test result or the terminal output.